res/WELCOME.md

```diff
@@ -2,7 +2,7 @@
 
 Hi @${username}, and welcome to GitGitGadget, the GitHub App to send patch series to the Git mailing list from GitHub Pull Requests.
 
-Please make sure that this Pull Request has a good description, as it will be used as cover letter.
+Please make sure that your Pull Request has a good description, as it will be used as cover letter (the commit message for your patch).
```

Comment:
I don't think that the addition "(the commit message for your patch)" is correct... the cover letter is the cover letter, it will be sent as a separate email, and the patch(es) will be sent as reply (or replies) to that cover letter.

Reply:
Ok. I think we can make that clearer as well, as I was under the wrong impression.